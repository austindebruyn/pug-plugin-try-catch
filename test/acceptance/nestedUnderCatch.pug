.foo
  try
    h1= state.missingProperty.name
  catch
    try
      h1= state.missingProperty.abbr
    catch
      h1 No name or abbreviation!
