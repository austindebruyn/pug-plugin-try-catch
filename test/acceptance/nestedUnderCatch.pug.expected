<div class="foo"><h1>No name or abbreviation!</h1></div>
